src/modules/notifications/Notifier.ts

```diff
@@ -14,24 +13,36 @@ export default class Notifier {
         sound = null,
         setting = null,
         image = null,
+        htmlLessMessage = null,
     }: {
         message: string;
         type?: NotificationOption;
         title?: string;
         timeout?: number;
         time?: string;
         sound?: Sound;
-        setting?: BooleanSetting;
+        setting?: NotificationSetting;
         image?: string;
+        htmlLessMessage?: string;
     }): void {
         $(document).ready(() => {
             // If we have sounds enabled for this, play it now
             if (sound) {
                 sound.play();
             }
 
+            if (setting && setting.desktopNotification.value && Notification.permission === 'granted') {
+                const desktopNotification = new Notification(title, {
+                    body: htmlLessMessage ?? message,
+                    icon: image,
+                });
```

Comment:
Should we also respect the sound setting here?
```suggestion
                const desktopNotification = new Notification(title, {
                    body: htmlLessMessage ?? message,
                    icon: image,
                    silent: (check if sound disabled)
                });
```
Maybe it should also be silent if they have game and desktop notifications on, as the in game sound should still play?

Reply:
So, right now the sound does not care, if you have the notification on or not. I did not know if it was intended, or not, but i did not change that part.

Firefox does not have sounds on their notifications (so i did not notice it, during development) and Chromes sounds are waaaaaay worse than ours... So i just disabled sounds... If they have sound enabled, it will now play the in game, no matter how their notification-settings are set. I think it's the best solution